src/library/scala/collection/immutable/LazyList.scala

```diff
@@ -164,11 +170,11 @@ import scala.runtime.Statics
   *  }}}
   *
   *  - The fact that `tail` works at all is of interest.  In the definition of
-  *  `fibs` we have an initial `(0, 1, LazyList(...))` so `tail` is deterministic.
```

Comment:
this comment/note seems out of place. we haven't been talking about `fibs` for a while

Reply:
Good point. Really fixing this well would probably require bigger edits than I want to make it right now, but I will apply a band-aid.